### PR TITLE
Handle delete from MS

### PR DESCRIPTION
**Pull request by stulees**

### CODE REVIEW

File: src/messaging/delete/delete.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    if (!gcsValidator.validateFilepath(filePath)) {
+      return Promise.reject(new Error("Invalid delete message"));
+    }
+
+    return db.fileMetadata.delete(filePath)
+      .then(db.owners.delete(filePath))
+      .then(db.watchlist.delete(filePath))
+      .then(()=>{
+        broadcastIPC.broadcast("FILE-DELETE", filePath);
```
Comment (tejohnso) [inline src/messaging/delete/delete.js]: Should we go with the FILEUPDATE topic and with a type of "DELETE" as we do for MS messages?

File: src/messaging/messaging.js [hunk 2]
```diff
@@ -25,6 +26,13 @@ function handleMSFileUpdate(message) {
         console.log(err);
       });
   }
+
+  if (message.type === "delete") {
```
Comment (tejohnso) [inline src/messaging/messaging.js]: Should compare against toUpperCase here

File: src/messaging/delete/delete.js (hunk 1 above)
Comment (stulees) [reply src/messaging/delete/delete.js]: @tejohnso Yeah I like that better, I'll make the change, thanks.

**Comment by stulees** [comment]
@tejohnso Please review last 2 commits. Instead of providing `type`, it's providing a `status` of "DELETED". I think it's more suitable as `status` is provided in other FILE-UPDATE messages.

**Pull request merged**